.eslintrc

```diff
@@ -1,2 +1,8 @@
----
-  extends: braintree/client
+{
+  "extends": "braintree/client",
+  "env": {
```

Comment:
The env is already defined in the config https://github.com/braintree/eslint-config/blob/68b1e9d94fc351863f6be4cd1a8f232cebf40dd8/client.js#L5-L8

We can't do es6, because all the Braintree modules need to be es5 compatible.

Is there any reason to change this from yaml format to a json  object?

Reply:
Apologies, didn't realise it had to be es5. Reverted this file back